easy-ui-react/src/ProductLayout/_mixins.scss

```diff
@@ -0,0 +1,7 @@
+@use "../styles/common" as *;
+
+@mixin breakpoint-large-screen {
+  @include breakpoint-lg-up {
+    @content;
+  }
+}
```

Comment:
wanted to consolidate this breakpoint into one place for easy changing in the future